setup.py

```diff
@@ -121,20 +121,20 @@ def run(self):
         update_package_data(self.distribution)
 
 version_ns = {}
-with open(os.path.join(here, 'yt-pycanvas', '_version.py')) as f:
+with open(os.path.join(here, 'yt_pycanvas', '_version.py')) as f:
     exec(f.read(), {}, version_ns)
 
 setup_args = {
-    'name': 'yt-pycanvas',
+    'name': 'yt_pycanvas',
     'version': version_ns['__version__'],
     'description': 'A Custom Jupyter Widget Library',
     'long_description': LONG_DESCRIPTION,
     'include_package_data': True,
     'data_files': [
         ('share/jupyter/nbextensions/yt-jscanvas', [
-            'yt-pycanvas/static/extension.js',
-            'yt-pycanvas/static/index.js',
-            'yt-pycanvas/static/index.js.map',
+            'yt_pycanvas/static/extension.js',
```

Comment:
I noticed that you changed the name to yt_pycanvas from yt-pycanvas. The line above this one has the javascript package that we named for the canvas widget `yt-jscanvas`. Should we be consistent with the underscoring vs. hyphen, or is there a best practice with using one over the other?

Reply:
Ah!  Good question.  I think it's a big deal with python, but not with javascript, maybe?  I was unable to get the library to import unless I changed it.

Reply:
Weird! Well, maybe down the line I can update the javascript lib so they have consistent naming conventions.